code/modules/deathmatch/deathmatch_controller.dm

```diff
@@ -76,6 +76,9 @@
 		return
 	switch (action)
 		if ("host")
+			if(!(GLOB.ghost_role_flags & GHOSTROLE_MINIGAME))
+				to_chat(usr, span_warning("Deathmatch has been temporarily disabled by admins."))
```

Comment:
I think this should be a tgui-alert rather than a to-chat, chat is likely not where they're looking when using the menu

(or for bonus points the ui itself should have a dimmer or something, but you don't have to do that)